config/config.go

```diff
@@ -63,12 +71,36 @@ func Get() Config {
 		printResponse = (val == "1" || val == "true")
 	}
 
+	printResponseBody := false
+	if val, exists := os.LookupEnv("print_response_body"); exists {
+		printResponseBody = (val == "1" || val == "true")
+	}
+
+	printSync := false
+	if val, exists := os.LookupEnv("print_sync"); exists {
+		printSync = (val == "1" || val == "true")
+	}
+
+	asyncFunctionInvocation := true
+	if val, exists := os.LookupEnv("async_function_invocation"); exists {
+		asyncFunctionInvocation = (val == "1" || val == "true")
+	}
+
+	topicAnnotationDelimiter := ","
+	if val, exists := os.LookupEnv("topic_annotation_delimiter"); exists {
+		topicAnnotationDelimiter = val
```

Comment:
Similarly here, faas-provider has a string reader with default fallback https://github.com/openfaas/faas-provider/blob/d6579bdcf7c85f4d01f398d65ea0cab37e9633d0/types/read_config.go#L69

Reply:
I'm keeping it consistent with the previous code.